index.html

```diff
@@ -5,11 +5,43 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Quiz App</title>
+    <link rel="stylesheet" href="styles.css">
 </head>
 <body>
     <header>
         <h1>Quiz App</h1>
     </header>
+    <nav>
+        <ul class="navbar">
+            <li><a href="#" id="home">🏠</a></li>
+            <li><a href="#" id="bookmark">💜</a></li>
+            <li><a href="#" id="profile">👤</a></li>
+        </ul>
+    </nav>
+
+    <main>
+        <section class="question_card">
```

Comment:
in a section you should always have a heading.

The question of your cards could for instance be an `h2`